aiida/plugins/entry_point.py

```diff
@@ -213,92 +217,75 @@ def load_entry_point(group, name):
     return loaded_entry_point
 
 
-def get_entry_point_groups():
+def get_entry_point_groups() -> Set[str]:
```

Comment:
hm... should call perhaps also be cached?

I'm wondering a bit now what the policy here is for which function calls to cache and which not - I guess any expensive call to `eps()` should be cached.
It would be good to document this in the module docstring

Reply:
yeh I was just using the ones that were already there. I would think though that maybe `impotlib_metadata` would already do its own caching, although I guess it can't hurt 🤷

Reply:
> I would think though that maybe `impotlib_metadata` would already do its own caching

I see, can you check?